index.html

```diff
@@ -1176,8 +938,10 @@ <h2>PNG image</h2>
           all others are fully opaque. If the alpha channel is not represented in this way, all pixels are fully opaque.
         </li>
 
-        <li>Greyscale: each pixel consists of a single sample: grey. The alpha channel may be represented by a single greyscale
-        pixel value, similar to the previous case. If the alpha channel is not represented in this way, all pixels are fully
+        <!-- Maintain a fragment named "3greyscale" to preserve incoming links to it -->
+        <li>Greyscale: each pixel consists of a single <dfn  id="3greyscale">grey sample</dfn>, which represents overall <a>luminance</a> (on a scale
+        from black to white). The alpha channel may be represented by a single <a>grey sample</a> value: matching pixels are fully
+        transparent, and all others are fully opaque. If the alpha channel is not represented in this way, all pixels are fully
```

Comment:
This wording is unclear to me.
It sounds like you're saying if the gray scale pixel has the value 86 and the alpha channel also has 86, this pixel will be transparent. Otherwise, this pixel is fully opaque.

Reply:
See if this works better. If so, we should probably change the similar sentence under _Truecolour_.

Reply:
I think that wording is better, yeah. Thank you :)
And good catch on _Truecolour_ needing an update, too.